app/org/sagebionetworks/bridge/validators/StudyValidator.java

```diff
@@ -283,7 +287,7 @@ private void validateEmailTemplate(Errors errors, EmailTemplate template, String
             }
             if (StringUtils.isBlank(template.getBody())) {
                 errors.rejectValue("body", "cannot be blank");
-            } else {
+            } else if (templateVariables.length > 0){
```

Comment:
Do we have any test cases where there are no template vars? Should we add unit tests for this case? (Both email and SMS.)